src/net_processing.cpp

```diff
@@ -1109,35 +1108,35 @@ void static ProcessGetBlockData(CNode* pfrom, const Consensus::Params& consensus
     LOCK(cs_main);
     const CBlockIndex* pindex = LookupBlockIndex(inv.hash);
     if (pindex) {
```

Comment:
you should return if pindex is not defined

Reply:
indeed good catch